firebase.go

```diff
@@ -200,3 +200,23 @@ func (r *Reference) Delete() error {
 
 	return nil
 }
+
+func (r *Reference) Child(path string) *Reference {
+	url := r.url
+
+	// Append path.
+	if url[len(url)-1] == '/' {
+		url = url[:len(url)-1]
+	}
+	if path[0] == '/' {
+		url += path
+	} else {
+		url += "/" + path
+	}
```

Comment:
I guess it can be replaced by [path.Join](https://golang.org/pkg/path/#Join)

cf) https://play.golang.org/p/OaN70vvjsj